.github/workflows/generate_schema.yml

```diff
@@ -70,29 +88,41 @@ jobs:
     - name: checkout latest release
       run: |
         git checkout $SWTAG
-        
-    - name: checkout commercial plugin
+
+    - name: checkout ${{ inputs.plugin_name }} plugin
       env:
         DEPLOY_USER: ${{ secrets.DEPLOY_USER }}
         DEPLOY_TOKEN: ${{ secrets.DEPLOY_TOKEN }}
+        DIGITALSALESROOM_DEPLOY_TOKEN: ${{ secrets.DIGITALSALESROOM_DEPLOY_TOKEN }}
```

Comment:
Seems like we can reuse the `DEPLOY_USER` and `DEPLOY_TOKEN` instead?

Reply:
We always get one user + token per plugin repo. I would rather rename them to the specific use case later on.

COMMERCIAL_DEPLOY_TOKEN etc.

As for the assignment, not sure if this makes a difference. The scopes  in github do irritate me sometimes of the env / vars etc. :smile: